src/elements/OcSidebar.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <div :class="$_ocSidebar_classes">
+    <oc-button
+      v-if="fixed"
+      class="oc-sidebar-button-close"
+      variation="raw"
+      @click.native="$_ocSidebar_buttonClose_click" 
+      :aria-label="closeButtonLabel"
+    >
+      <oc-icon name="close" aria-hidden="true" />
+    </oc-button>
+    <router-link to="/" class="oc-sidebar-logo">
+      <oc-img
+        v-if="logoImg"
+        key="logo-image"
+        :src="logoImg"
+        :alt="productName"
+        class="oc-sidebar-logo-img"
+      />
+      <span v-else key="product-name" class="oc-sidebar-logo-text" v-text="productName" />
+    </router-link>
+    <div v-if="$slots.upperContent" class="oc-sidebar-upper-content">
+      <!-- @slot Content above the navigation block -->
+      <slot name="upperContent" />
+    </div>
+    <nav :class="[{ 'uk-margin-bottom' : $slots.footer },  'oc-sidebar-nav']">
+      <ul>
+        <oc-sidebar-nav-item
+          v-for="item in navItems"
+          :key="item.route.path"
+          :icon="item.icon || item.iconMaterial"
+          :active="item.active"
+          :target="item.route.path"
+          class="oc-sidebar-nav-item"
+        >
+          {{ item.name }}
+        </oc-sidebar-nav-item>
+      </ul>
+    </nav>
+    <div v-if="$slots.footer" class="oc-sidebar-footer">
+      <!-- @slot Footer of the sidebar -->
+      <slot name="footer" />
+    </div>
+  </div>
+</template>
+
+<script>
+import OcSidebarNavItem from "./OcSidebarNavItem.vue"
+import OcImg from "./OcImage.vue"
+import OcButton from "./OcButton.vue"
+import OcIcon from "./OcIcon.vue"
+
+export default {
+  name: "oc-sidebar",
+  status: "review",
+  released: "unreleased",
+
+  components: {
+    OcSidebarNavItem,
+    OcImg,
+    OcButton,
+    OcIcon
+  },
+
+  props: {
+    /**
+     * Source of the logo image
+     */
+    logoImg: {
+      type: String,
+      required: false
+    },
+    /**
+     * Name of the product where the sidebar is used
+     */
+    productName: {
+      type: String,
+      required: true
+    },
+    /**
+     * Navigation items of the product
+     */
+    navItems: {
+      type: Array,
+      required: true
+    },
+    /**
+     * Asserts whether the sidebar's position is fixed
+     */
+    fixed: {
+      type: Boolean,
+      required: false,
+      defult: false
+    },
+    /**
+     * Accessibility label of the close button
+     */
+    closeButtonLabel: {
+      type: String,
+      required: false,
+      default: "Close navigation menu"
+    }
+  },
+
+  computed: {
+    $_ocSidebar_classes() {
+      const classes = ["oc-sidebar"]
+
+      if (this.fixed) {
+        classes.push("oc-sidebar-fixed")
+      }
+
+      return classes
+    }
+  },
+
+  methods: {
+    $_ocSidebar_buttonClose_click() {
+      /**
+       * The user clicked on the close button
+       */
+      this.$emit("closed")
```

Comment:
Emitted event should be `close` in order to be aligned with other event names in ODS